PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -322,3 +325,13 @@ def miniAOD_customizeAllMC(process):
     miniAOD_customizeCommon(process)
     miniAOD_customizeMC(process)
     return process
+
+def miniAOD_forPhase2(process):
+    print "Switching off electron and photon IDs (not yet working for Phase2 era)"
+    process.patElectrons.addElectronID = False
+    process.patElectrons.electronIDSources = cms.PSet()
+    process.patPhotons.addPhotonID = False 
+    process.patPhotons.photonIDSources = cms.PSet()
```

Comment:
+1 for making a miniAOD era

given how PAT configs are setup, it looks like all era changes will still be stuck in a master file